### PR TITLE
zh-translation: /docs/ops/common-problems/validation/index.md

### PR DESCRIPTION
**Pull request by Lovnx**
https://github.com/servicemesher/istio-official-translation/issues/1240

### CODE REVIEW

File: content/zh/docs/ops/common-problems/validation/index.md [hunk 1]
```diff
@@ -147,10 +142,8 @@ webhooks:
     - tracespans
 {{< /text >}}
 
-If the `validatingwebhookconfiguration` doesn’t exist, verify the
-`istio-galley-configuration` `configmap` exists. `istio-galley` uses
-the data from this configmap to create and update the
-`validatingwebhookconfiguration`.
+如果 `validatingwebhookconfiguration` 不存在，那就验证
+`istio-galley-configuration` `configmap` 是否存在。`istio-galley` 使用 configmap 的数据来创建或更新 `validatingwebhookconfiguration`.
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/common-problems/validation/index.md]:
```suggestion
`istio-galley-configuration` `configmap` 是否存在。`istio-galley` 使用 configmap 的数据来创建或更新 `validatingwebhookconfiguration`。
```

File: content/zh/docs/ops/common-problems/validation/index.md [hunk 2]
```diff
@@ -9,18 +9,13 @@ aliases:
     - /zh/docs/ops/troubleshooting/validation
 ---
 
-## Seemingly valid configuration is rejected
+## 看似有效的配置不生效
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/common-problems/validation/index.md]: 所有标题需要加锚点

**Comment by tanjunchen** [comment]
@malphi leaving a comment that contains only @googlebot I consent on #5823 Thanks

**Pull request merged**